### PR TITLE
Display and edit translations for workflow contents

### PR DESCRIPTION
**Pull request by eatyourgreens**

### CODE REVIEW

File: src/components/WorkflowContents.jsx [hunk 1]
```diff
@@ -13,12 +13,17 @@ options.log = (test) => {
...
   const { contents } = props;
   const workflow_contents = contents.original || { strings: {} };
+  const translation = contents.translation || { strings: {} };
   const strings = [];
   const keys = workflow_contents.strings ? Object.keys(workflow_contents.strings) : [];
-  keys.map(key => strings.push(<p key={key}><b>{key}</b> {workflow_contents.strings[key]}</p>));
+  keys.map((key) => {
+    strings.push(<p data-translation-key="strings" data-translation-subkey={key} key={key}><b>{key}</b> {workflow_contents.strings[key]}</p>);
+    strings.push(<p key={`translation-${key}`}><b>Translation</b> {translation.strings[key]}</p>);
+  });
```
Comment (simoneduca) [inline src/components/WorkflowContents.jsx]: The above lines have become difficult to read.

Comment (eatyourgreens) [inline src/components/WorkflowContents.jsx]: Sorry, that's what the field is actually called in Panoptes.

File: src/components/WorkflowContents.jsx [hunk 2]
```diff
@@ -13,12 +13,17 @@ options.log = (test) => {
 function WorkflowContents(props) {
   const { contents } = props;
   const workflow_contents = contents.original || { strings: {} };
+  const translation = contents.translation || { strings: {} };
```
Comment (eatyourgreens) [inline src/components/WorkflowContents.jsx]: Sorry, that's what the field is actually called in Panoptes.

File: src/components/WorkflowContents.jsx (hunk 1 above)
Comment (eatyourgreens) [inline src/components/WorkflowContents.jsx]: Yep. At some point, the resource views will need some basic design work.

**Pull request merged**